game_event/views.py

```diff
@@ -134,7 +137,8 @@ def remove_from_event(request, id):
         return render(request, 'game_event/result.html', {'event_exists': False})
     player = Player.objects.get(user=request.user)
     GameEventPlayer.objects.get(game_event=event, player=player).delete()
-    return render(request, 'game_event/result.html', {'event_exists': True})
+    event_exists = event.delete_event_if_no_players()
+    return render(request, 'game_event/result.html', {'event_exists': not event_exists})
```

Comment:
Something still not working for me here:

- `delete_event_if_no_players` returns true if it deleted the event, so the event does not exist.
- Then we assign `not event_exists` to another `event_exists`.

This suggests that something is not named right.